var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -62,3 +64,5 @@ class PyPytest(PythonPackage):
     depends_on('py-importlib-metadata@0.12:', when='@4.6:5.0', type=('build', 'run'))
     depends_on('py-importlib-metadata@0.12:', when='@5.1: ^python@:3.7', type=('build', 'run'))
     depends_on('py-wcwidth', when='@4.5:', type=('build', 'run'))
```

Comment:
This dep doesn't seem to be needed anymore

Reply:
* I added a version constraint for `py-wcwith`
* `py-importlib-metada` is still neeeded if python < 3.8 so I changed the constraint to `python@:3.7.999`